v3/integration/README.md

```diff
@@ -332,3 +332,32 @@ Once you're confident the observed results match expectations you can add the
 new lint results to the expected data by running the integration tests with
 `-overwriteExpected` and committing the updated config file along with your new
 lint. Nice work!
+
+Generating Test Certificates
+-----------------
```

Comment:
This feels like something better suited to CONTRIBUTING.md than v3/integration/README.md.

It feels confusing to have this here because the integration test data corpus is entirely a static set of real world certificates from Censys, no special snowflakes.

Reply:
Coming back around, ditto for the script itself: can it live somewhere other than `v3/integration`? It isn't used by integration tests or run in CI.